src/System/OS.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Dock\System;
+
+class OS
```

Comment:
Can you rename that `OS` class with a better name ? For instance `OperatingSystemDetector` or even something better :)